crates/cast/src/helpers/response_structs.rs

```diff
@@ -36,6 +36,11 @@ pub struct AccountAddResponse {
     pub add_profile: String,
 }
 
+#[derive(Serialize)]
+pub struct AccountDeleteResponse {
+    pub result: String,
```

Comment:
Should there also be a notification tied to `--delete-profile`? i.e when it's not passed, maybe there should be a notification that it has been left in the Scarb.toml - and when passed - that it has been deleted from there?